### ISSUE TITLE
Git submodule is a private repo

### ISSUE THREAD

**Issue by AntoineRondelet** [ekiden/contract-evm#1]
I tried to play around with your contract EVM repo, but unfortunately I couldn't run the `git submodule` command successfully. Here's an extract of the error I have:

```
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:sunblaze-ucb/ekiden.git'
```

Thus, I guess the `sunblaze-ucb/ekiden` repo is private. Any chance to make it open source, so that we can try to run the contract-evm code ?

Cheers,

Antoine

**Comment by polymorpher** [comment]
Yes, in addition to that I am also looking for cryptokitties code mentioned in the paper https://arxiv.org/pdf/1804.05141.pdf
